distributed/utils_test.py

```diff
@@ -766,12 +783,6 @@ def cluster(
         else:
             client.close()
 
-    start = time()
-    while any(proc.is_alive() for proc in ws):
```

Comment:
procs are closed, i.e. `is_alive` is not possible anymore. In fact, it even raises an exception since you can't even interact with the proc object anymore after it is being closed